core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -394,6 +394,14 @@ static <T> StreamMessage<T> aborted(Throwable cause) {
         return new AbortedStreamMessage<>(cause);
     }
 
+    /**
+     * Creates a new {@link StreamMessageWriter} that publishes the objects written via
+     * {@link StreamWriter#write(Object)}.
+     */
+    static <T> StreamMessageWriter<T> streaming() {
+        return new DefaultStreamMessage<>();
```

Comment:
So you want to deprecate this class in the follow-up PR. Is it correct?

Reply:
<img width="989" alt="image" src="https://user-images.githubusercontent.com/34854527/220832106-9c0081d2-4264-4a80-8c25-15a5417013f5.png">

```
AS-IS: StreamMessageWriter<> writer = new DefaultStreamMessage()<>;
TO-BE: StreamMessageWriter<> writer = StreamMessage.streaming();
```

aha I just want to replace above `new DefaultSTreamMessage<>();` usage to `StreamMessage.streaming()` ~!

Reply:
> you want to deprecate this class

hmm.. deprecate whole DefaultStreamMessage class? or `new DefaultStreamMessage<>()` consturctor?

Reply:
I mean deprecating the whole class because it doesn't need to be exposed to public. Isn't it correct?

Reply:
> I mean deprecating the whole class because it doesn't need to be exposed to public

aha~ I understood. right! I'll deprecate whole `DefaultStreamMessage` on next PR, to not expose to public~!

Reply:
✅ clean-up PR: https://github.com/line/armeria/pull/4717